src/OrchardCore.Modules/OrchardCore.ContentFields/Views/DateField-Custom.Edit.cshtml

```diff
@@ -0,0 +1,37 @@
+@model OrchardCore.ContentFields.ViewModels.EditDateFieldViewModel
+
+@using OrchardCore.ContentFields.Settings;
+@using OrchardCore.ContentManagement.Metadata.Models
+@using OrchardCore.ResourceManagement
+
+@inject IResourceManager ResourceManager
+
+@{
+    var settings = Model.PartFieldDefinition.Settings.ToObject<DateFieldSettings>();
+    AppendDatePickerScript();
+}
+
+@functions{
+    private void AppendDatePickerScript()
+    {
+        const string cultureSeparator = "-";
```

Comment:
I think you can use `CultureInfo.CurrentUICulture` properties to get the parent culture directly

Reply:
I was thinking about that before, but there are some cultures like Chinese are crazy 😂, it will requires a loop to get the root culture, so that's why I prefer this way